### PR TITLE
first interaction with arcade employee

### PR DESCRIPTION
**Pull request by abdulrahimsulthan**
First interaction with arcade employee done, Need a time to get some fun idea with games playing.

### CODE REVIEW

**Comment by abdulrahimsulthan** [comment]
#30 linked to this issue

File: src/interactions/map_arcade/arcade_employee.interaction.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { firstConversationArcade } from "../../constants";
+import { displayDialogue } from "../../utils";
```
Comment (r4pt0s) [inline src/interactions/map_arcade/arcade_employee.interaction.js:2]: first, please pull the latest changes from the main branch.
Then, replace the `displayDialoge`function import with the new function `displayDialogueWithCharacter` which you can find [here](https://github.com/zero-to-mastery/ZTM-Quest/blob/c6296090a3dc8b0130fdb72af1458d66d8e7a4cb/src/utils.js#L35) as a reference.
This new function accepts the character name as first parameter. The rest is the same.
You can get creative and give the aracde_employee a fancy name

**Review by r4pt0s — CHANGES_REQUESTED** [review]
Please have a look at the comments. There is a change necessary.
Thank you

File: src/interactions/map_arcade/arcade_employee.interaction.js (hunk 1 above)
Comment (abdulrahimsulthan) [reply src/interactions/map_arcade/arcade_employee.interaction.js]: Changes done.
src/interactions/map_start/jokeTeller.interaction.js I am not created this file it give some lint issues.

**Comment by abdulrahimsulthan** [comment]
I didnt touch this file it give some linting issues src/interactions/map_start/jokeTeller.interaction.js

**Comment by r4pt0s** [comment]
@abdulrahimsulthan I know. Can you just update the PR with the latest version of main?
The issue with the liniting got already resolved. Thank you :pray:

**Review by r4pt0s — CHANGES_REQUESTED** [review]
Could you please make the requested changes :pray:

**Comment by b-hexsoul** [comment]
please also checkout npcInteractionHandler function in handler.interactions.js

**Review by lorenzomiro — COMMENTED** [review]
Changes were made as of 10/7/2024

**Comment by r4pt0s** [comment]
@abdulrahimsulthan are you still working on it?

**Comment by r4pt0s** [comment]
Closed due to inactivity. @abdulrahimsulthan if still want to make these changes, please open a new pull request 🙏🏻

**Pull request closed without merging**